lucene/core/src/java/org/apache/lucene/index/DocumentsWriterPerThread.java

```diff
@@ -156,6 +158,9 @@ void abort() throws IOException{
   final BufferedUpdates pendingUpdates;
   final SegmentInfo segmentInfo;     // Current segment we are working on
   private boolean aborted = false;   // True if we aborted
+  private SetOnce<Boolean> flushPending = new SetOnce<>();
+  private volatile long lastCommittedBytesUsed = 0;
```

Comment:
`= 0` isn't needed -- it's Java's default already.

Reply:
👍